.github/ISSUE_TEMPLATE/docs_improvement.md

```diff
@@ -0,0 +1,20 @@
+---
+name: Docs Improvement
+about: Help us write better docs to catch common issues!
+title: ''
+labels: documentation, needs-triage
+assignees: ''
+---
+
+## What problem did the user encounter?
```

Comment:
The wording here feels like the issue should be written by someone who isn't the user. We're essentially asking the user to refer to _themselves_ as the user, which feels awkward. Additionally, the first two sections don't seem particularly useful. Maybe we only need a single section for documentation improvements?

(I added / tweaked a few things)

```
## How can Bevy's documentation be improved?

Provide a link to the documentation and describe how it could be improved. In what ways is it incomplete, incorrect, or misleading?

If you have suggestions on exactly what the new docs should say, feel free to include them here. Or alternatively, make the changes yourself and [create a pull request](https://bevyengine.org/learn/book/contributing/code/) instead.
```